LICENSE

```diff
@@ -199,3 +199,19 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+=======================================================================
+PyWren Subcomponents:
+
+The PyWren project contains subcomponents with separate copyright
+notices and license terms. Your use of the source code for the these
+subcomponents is subject to the terms and conditions of the following
+licenses.
+
+========================================================================
+BSD-style licenses
```

Comment:
Can you explicitly note that some files are BSD 2 clause and some BSD 3 clause ? They are broadly similar but its better to be precise.

Reply:
Good point! Done.